zuul.d/playbooks/upgrade/run-zed-2023.1.yml

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2024 VEXXHOST, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Upgrade test
+  hosts: all
+  become: true
+  tasks:
+    - name: Checkout to Zed branch
+      become: true
+      ansible.builtin.shell: |
+        sudo apt-get purge -y snapd
+        git stash
+        git checkout stable/zed
+      args:
+        chdir: "{{ zuul.project.src_dir }}"
+
+    - name: Run molecule with Zed branch
+      become: true
+      ansible.builtin.shell: |
+        export ATMOSPHERE_NETWORK_BACKEND=openvswitch
+        pip install --ignore-installed poetry
+        poetry install
+        poetry run molecule converge -s aio
+      args:
+        chdir: "{{ zuul.project.src_dir }}"
+
+    - name: Add rabbitmq_skip_spec_diff
+      ansible.builtin.lineinfile:
+        path: "{{ ansible_user_dir }}/{{ zuul.project.src_dir }}/molecule/aio/group_vars/all/molecule.yml"
+        insertafter: EOF
+        line: 'rabbitmq_skip_spec_diff: true'
```

Comment:
Is this necessary?  In my experience, only when you have a TTY it will block, but in CI or when there is automation, it skips right through.